<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Purchase</class>
 <widget class="QDialog" name="Purchase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="list_of_souvenir_scroll_area_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>319</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addToCart">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Stadium</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backToTripType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="subtotal_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="cart_scroll_area_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>319</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="removeFromCart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove from Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="cart">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="stadiumName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_stadium">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Stadium</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSouvenir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>List of Souvenirs</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
